tests/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ endif()
 
 add_subdirectory(utils)
 add_subdirectory(host_reference)
-enable_language(Fortran)
+# enable_language(Fortran)
```

Comment:
Why is Fortran disabled here?

Reply:
The CI initially did not have a fortran compiler in the build container. And tbh I don't know why it was here in the first place. We conditionally enable fortran in the top level CMakeLists.txt